### PR TITLE
feat: implemented functionality for help/about/privacy shopMobile

### PR DESCRIPTION
**Pull request by sunk0**
help-content
about-content
privacy-content

### CODE REVIEW

**Comment by sunk0** [comment]
merchant-about-us

File: packages/merchant-tablet-ionic/src/pages/+info/+about/about.html [hunk 1]
```diff
@@ -11,16 +11,11 @@ <h1 class="title">{{ 'ABOUT_VIEW.TITLE' | translate }}</h1>
...
+		class="pt-4"
+		*ngIf="useAboutHtml !== null"
+		[innerHTML]="useAboutHtml | safe: 'html'"
 	></div>
-	<h5
-		style="padding-left: 10px; padding-top: 20px; color: #888;"
-		*ngIf="aboutHtml !== null"
-	>
-		{{ 'ABOUT_VIEW.APP_VERSION' | translate }}: {{ appVersion }}
-	</h5>
```
Comment (evereq) [inline packages/merchant-tablet-ionic/src/pages/+info/+about/about.html:25]: @sunk0 why you removed that?????

**Comment by evereq** [comment]
@sunk0 please revert back things you removed

**Comment by sunk0** [comment]
@evereq reverted

**Comment by evereq** [comment]
@sunk0 ok, so I spend a bit more time on this and I think you still removed important feature that was supposed to be working before.

This is calls that was before to 2 functions:
```
this.userRouter.getPrivacy(this.store.userId, this.store.deviceId);
this.userRouter.getAboutUs(this.store.userId, this.store.deviceId);
```
As you can see, before what user see on the page depends on userId and deviceId. Now you added `languageId`, however you removed that "userId" and "deviceId" parameters........ That should be fixed, so it now probably should have 3 parameters: "userId","deviceId", "languageId". It's best to do it for all such methods which return static pages....

**Comment by evereq** [comment]
@sunk0 why do you call that `this.deviceRouter.updateLanguage(this.deviceId, this.selectedLanguage);` everywhere? What is the reason? Also, why you did not pass anymore language to all calls like 
```
this.userRouter.getPrivacy(this.store.userId, this.store.deviceId);
this.userRouter.getAboutUs(this.store.userId, this.store.deviceId);
```
?
We discussed and I suggested to add 3 parameters in such calls, but you still send 2... :(

**Comment by sunk0** [comment]
@evereq  now should be fine

**Comment by evereq** [comment]
@sunk0 I merged your changes, however, honestly - I think many of them was not really needed because you did not figure out how it was worked before in some aspects...

There was code like this:
```
switchMap((device) => this.watchedFiles.termsOfUse[device.language])
```
which you replaced now with
```
switchMap((device) => this.watchedFiles.termsOfUse[language])
```

So before IF device was set and come from DB for a user, it uses `device.language` field to figure out in which language to render pages. The issue was only when device record did not exist in DB yet (not sure when it could happen really... maybe before user register for example) and so in such cases previously you can't see pages (privacy, ToS, etc). Now you fixed it by always use currently selected language in UI etc, which will work for now, but at some point, we will need to revisit this again, etc...

Anyway, merged, let's move to other tasks...

**Pull request merged**